Dockerfile

```diff
@@ -57,5 +69,17 @@ RUN ./docker/build --backend
 
 RUN sha256sum /backend.wasm.gz
 
+
+ARG network="staging"
+ENV ENV=$network
+ENV DFX_NETWORK=$network
+
+RUN echo $ENV
+RUN echo $DFX_NETWORK
+
+RUN npm ci
+RUN npm run build
+
 FROM scratch AS scratch_backend
```

Comment:
Imho this should be renamed or split into a separate stage, see above.

Reply:
Its in a separate dockerfile now, so this no longer applies